### PR TITLE
update symlink for the release of v1.0.0-errata

### PR DESCRIPTION
**Pull request by sunghee-hwang**

### CODE REVIEW

File: v1.0.0-errata.html [hunk 1]
```diff
@@ -0,0 +1,7947 @@
...
+    <li data-md>
+     <p>Let the term "Audio Frames" mean the set of all <a data-link-type="dfn" href="#audio-frame-obu" id="ref-for-audio-frame-obu②④">Audio Frame OBU</a>s (for this <a data-link-type="dfn" href="#audio-element" id="ref-for-audio-element④④">Audio Element</a>) that have the same start timestamp. All Audio Frames in an <a data-link-type="dfn" href="#ia-sequence" id="ref-for-ia-sequence③⓪">IA Sequence</a> SHALL have the same number of <a data-link-type="dfn" href="#audio-frame-obu" id="ref-for-audio-frame-obu②⑤">Audio Frame OBU</a>s.</p>
+    <li data-md>
+     <p><a data-link-type="dfn" href="#parameter-block-obu" id="ref-for-parameter-block-obu①⑥">Parameter Block OBU</a>s MAY be associated with Audio Frames.</p>
+   </ul>
+   <center><img height="356" src="images/Immersive Audio Sequence with scalable channel audio (before OBU packing).png" style="width:100%; height:auto;" width="1622"></center>
```
Comment (cconcolato) [inline v1.0.0-errata.html:2808]: The links to images need to be fixed

**Comment by cconcolato** [comment]
Can you make just one PR? with this one and #804.

**Comment by sunghee-hwang** [comment]
> Can you make just one PR? with this one and #804.

I can add these files to #804 but I am not sure if those files will be reflected in this 'gh-pages' branch after merging them into the 'main' one.
When we merge something to the 'main' branch, it seems that the index.bs file is only reflected in this 'gh-pages' one.

File: v1.0.0-errata.html (hunk 1 above)
Comment (sunghee-hwang) [reply v1.0.0-errata.html:2808]: Done and now, all changes (including symlinks) for this PR are reflected in #804.

**Comment by cconcolato** [comment]
We should not use gh-pages anymore. We should be using GitHub actions. See https://github.com/AOMediaCodec/av1-isobmff/blob/main/.github/workflows/auto-publish.yml for example

**Comment by sunghee-hwang** [comment]
> We should not use gh-pages anymore. We should be using GitHub actions. See https://github.com/AOMediaCodec/av1-isobmff/blob/main/.github/workflows/auto-publish.yml for example

I replaced the auto-publish.yml in iamf with the one in av1-isobmff. (See #804)

**Review by tdaede — APPROVED** [review]

**Pull request merged**